ResourceStatusSystem/Client/ResourceStatus.py

```diff
@@ -228,15 +254,23 @@ def __setCSElementStatus( self, elementName, elementType, statusType, status ):
     Sets on the CS the Elements status
     """
 
+    # DIRAC doesn't store the status of CEs nor FTS in the CS, so here we can just do nothing
+    if elementType in ('CE', 'FTS'):
+      return S_OK()
+
+    # If we are here it is because elementType is either 'StorageElement' or 'Catalog'
     statuses = self.rssConfig.getConfigStatusType( elementType )
     if statusType not in statuses:
       gLogger.error( "%s is not a valid statusType" % statusType )
       return S_ERROR( "%s is not a valid statusType: %s" % ( statusType, statuses ) )
 
-    csAPI = CSAPI()
-
-    cs_path = "/Resources/" + elementType
+    if elementType == 'StorageElement':
+      cs_path = "/Resources/StorageElements"
+    elif elementType == 'Catalog':
+      cs_path = "/Resources/FileCatalogs"
+      statusType = 'Status'
```

Comment:
This won't be changed here, anyway. I made 2 more tasks for implementing that. The location shown in the code is present in LHCb CS, but it may be completely outdated.